wicket-auth-roles/pom.xml

```diff
@@ -24,7 +24,7 @@
 		<relativePath>../pom.xml</relativePath>
 	</parent>
 	<artifactId>wicket-auth-roles</artifactId>
-	<packaging>bundle</packaging>
+	<packaging>jar</packaging>
```

Comment:
Does this break the OSGi support ?!
CC @mattrpav

Reply:
As far as I know, it shouldn't. A jar is an OSGi bundle, when it's manifest contains OSGi related information, but it's still a jar. The documentation for the maven-bundle-plugin even describes this setup as a valid solution: https://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html#_adding_osgi_metadata_to_existing_projects_without_changing_the_packaging_type

Reply:
Yes, type should be ‘bundle’ to activate the bundle plugin when generating the jar. 

ActiveMQ had changes to support reproducible builds and maintained the bundle plugin. 

I recommend we should know exactly what is the miss or change needed to support reproducible builds, and not just change the project type to jar.

Reply:
@mattrpav the bundle plugin is activated by binding one of its goals to the maven execution phase (in this case `manifest` in `process-classes`). This will generate the MANIFEST.MF, which will then be used by the maven-jar-plugin. I can see the correct MANIFEST.MF in the produced jar.

The problem with the maven-bundle-plugin is that it generates jars without timestamps on its entries. This is valid (and considered reproducible), but very inconvenient. There is no way to change this when using packaging `bundle`. I think the produced jars are fine. They do contain the correct MANIFEST.MF. However, we do not use OSGi, so I have no way of testing it.

Reply:
@papegaaij the bundle plugin can add any field to the MANIFEST.MF. What field are you looking for in the MANIFEST.MF?

Reply:
@mattrpav This change is to get correct timestamps on files (also called entries) in the jar files. The maven-bundle-plugin does not set these timestamps at all, causing them to revert to the jar-epoch (02-01-1980). The maven-jar-plugin honors `project.build.outputTimestamp`. If it also gives us reproducible builds, that's a bonus but not the reason for this change.

We do however not want to break OSGi support. However, none of the Wicket core developers uses OSGi, so this is very hard for us to test. With this change, the produced jars *should* still be valid OSGi bundles, but we have no good way to verify this. It be really great of you could check this.

As far as I know OSGi, the only thing that makes a normal jar an OSGi bundle is a set of entries in the MANIFEST.MF. These all seem unaffected by this change (I compared before and after), so I think the change is correct. But maybe you can point to something else that must be in place for an OSGi bundle?